src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -252,13 +252,9 @@ public ISecureDataFormat<AuthenticationProperties> StateDataFormat
             {
                 return _stateDataFormat;
             }
+            [param: NotNull]
```

Comment:
Same remark as above, these properties will all be converted to automatic properties :smile: